hydra-main/src/main/java/com/addthis/hydra/query/web/HttpQueryCallHandler.java

```diff
@@ -80,6 +80,7 @@ public static ChannelFuture handleQuery(ChannelHandler queryToQueryResultsEncode
         }
         query.setParameterIfNotYetSet("allocator", kv.getValue("allocator"));
         query.setParameterIfNotYetSet("allowPartial", kv.getValue("allowPartial"));
+        query.setParameterIfNotYetSet("tasks", kv.getValue("tasks"));
```

Comment:
If it's going to get first class treatment in the UI, maybe we should move this statement up with the other A listers. And/Or use a real field rather than a dynamic parameter.